scripts/jest/setupTests.www.js

```diff
@@ -18,6 +18,8 @@ jest.mock('shared/ReactFeatureFlags', () => {
   wwwFlags.enableUserTimingAPI = defaultFlags.enableUserTimingAPI;
   wwwFlags.disableJavaScriptURLs = defaultFlags.disableJavaScriptURLs;
   wwwFlags.enableDeprecatedFlareAPI = defaultFlags.enableDeprecatedFlareAPI;
+  wwwFlags.disableModulePatternComponents =
```

Comment:
@gaearon To remove this, what we need to do is find all the tests that rely on module style components and wrap them in a feature flag check.